File: app/src/main/java/com/example/practicals/java_practicals/IfForWhile.java
package com.example.practicals.java_practicals;

public class IfForWhile {
    public static void main(String args[]) {
        int num = 10, i;
        System.out.print("List of even numbers from 1 to 10 " + "\n");
        for (i = 1; i <= num; i++) {
            if (i % 2 == 0) {
                System.out.print(i + " ");
            }
        }

        System.out.print("\n"+"List of odd numbers from 1 to 10 " + "\n");
        i = 0;
        while(i <= num) {
            if (i % 2 != 0) {
                System.out.print(i + " ");
            }
            i++;
        }
    }
}

File: app/src/main/java/com/example/practicals/kotlin_practicals/IfForWhile.kt
package com.example.practicals.kotlin_practicals

class IfForWhile {
}
fun main() {
    var num:Int = 10
    var j:Int = 1
    print("even ")
    for (i in 1..num)
        if(i%2==0)print("$i ")
    println()

    print("odd ")
    var i:Int = 1
    while(i<num) {
        if(i%2!=0)print("$i ")
        i++
    }
}